src/ui/public/kuery/lib/functions/and.js

```diff
@@ -0,0 +1,46 @@
+import * as ast from '../ast';
+import { nodeTypes } from '../node_types';
+
+export function buildNodeParams(children, serializeStyle = 'implicit') {
+  return {
+    arguments: children,
+    serializeStyle
+  };
+}
+
+export function toElasticsearchQuery(node, indexPattern) {
+  const children = node.arguments || [];
+
+  return {
+    bool: {
+      filter: children.map((child) => {
+        if (child.type === 'literal') {
+          child = nodeTypes.function.buildNode('is', '*', child.value);
```

Comment:
I don't think so, that would alter the AST built from query string provided by the user. Then when we call `toKueryExpression` on this node we won't know if the original query string contained an explicit `is` function, or just a literal.

In my experience so far, it seems best to read the query string as-is with minimal "interpretations" at parse time and leave things like interpreting the meaning of a literal argument to the individual serializers.